skimage/transform/radon_transform.py

```diff
@@ -124,8 +124,48 @@ def _sinogram_circle_to_square(sinogram):
     pad_width = ((pad_before, pad - pad_before), (0, 0))
     return util.pad(sinogram, pad_width, mode='constant', constant_values=0)
 
+def iradon_workspace(theta, output_size, circle=False, full=True):
+    """
+    Generate workspace needed for iradon().
+
+    This allows work arrays to be calculated once for an output size and set
+    of angles, useful for multiple calls to iradon() for the same geometry.
 
-def iradon(radon_image, theta=None, output_size=None,
+    Parameters
+    ----------
+    theta : array_like, dtype=float, optional
+        Reconstruction angles (in degrees). Default: m angles evenly spaced
+        between 0 and 180 (if the shape of `radon_image` is (N, M)).
+    output_size : int
+        Number of rows and columns in the reconstruction.
+    circle : boolean, optional
+        Assume the reconstructed image is zero outside the inscribed circle.
+        Also changes the default output_size to match the behaviour of
+        ``radon`` called with ``circle=True``.
+    full : boolean, optional
+        Whether to generate full workspace (default) or only the
+        grid data (for older behavior, smaller memory usage).
+    Returns
+    -------
+    xpr, ypr, thw : tuple of ndarrays
```

Comment:
Functions with more than one return value always return a tuple, each one of those values _is_ an ndarray so I don't think the `tuple of` should be there.